### PR TITLE
forms: add configurable url for duplicate validator

### PR DESCRIPTION
**Pull request by ammirate**
Signed-off-by: Antonio Cesarano <cesarano2607@gmail.com>

## Checklist:

- [x] I have all the information that I need (if not, move to `RFC` and look for it).
- [x] I linked the related issue(s) in the corresponding commit logs.
- [x] I wrote [good commit log messages](https://github.com/torvalds/subsurface-for-dirk/blob/5f15ad5a86ada3c5e574041a5f9d85235322dabb/README#L92-L119).
- [x] My code follows the code style of this project.
- [x] I've added any new docs if API/utils methods were added.
- [x] I have updated the existing documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by ammirate** [comment]
ci test please

File: inspirehep/modules/forms/validators/simple_fields.py [hunk 1]
```diff
@@ -142,7 +142,10 @@ def inspirehep_duplicated_validator(inspire_query, property_name, collections=No
     Needs to be wrapped in a function with proper validator signature.
     """
     if does_exist_in_inspirehep(inspire_query, collections):
-        url = "http://inspirehep.net/search?" + urlencode({'p': inspire_query})
+        url = "{0}?{1}".format(
+            current_app['LEGACY_MATCH_ENDPOINT'],
+            urlencode({'p': inspire_query})
+        )
```
Comment (david-caro) [inline inspirehep/modules/forms/validators/simple_fields.py]: you are missing also on line 76

Comment (david-caro) [reply inspirehep/modules/forms/validators/simple_fields.py]: it should be `current_app.config['LEGACY_MATCH_ENDPOINT']` also

Comment (ammirate) [reply inspirehep/modules/forms/validators/simple_fields.py]: yeah, being in a hurry...

**Comment by bavaria95** [comment]
ci test please

**Pull request merged**